readme.md

```diff
@@ -26,7 +41,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+We use jQuery to target the form fields so we can then use them in the POST request
 ```
```

Comment:
:+1:

Also need an event listener so that this whole process can be triggered.